<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ip</class>
 <widget class="QWidget" name="Ip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>648</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" rowspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>当前网络名称：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>序号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>名称</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>要修改的网络名称：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_name">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_queding">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>Ip地址：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_ip">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>子网掩码：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_subnet">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>新Ip地址：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_ip_ipnew">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>192.168.10.99</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="pushButton_changeip">
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>更改Ip</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>17</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>手动更改IP地址：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_auotoip">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>恢复为自动获取IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_status">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
